### ISSUE TITLE
Add support for screen reading software

### ISSUE THREAD

**Issue by jfayre** [home-assistant/home-assistant-polymer#271]
I am blind and use a screen reader. The current Home Assistant UI is not accessible to people using screen reading technology. 
Please make the UI compliant with Web Content Accessibility Guidelines (WCAG) 2.0 specs.
As a start, I would recommend labeling the buttons on the UI. All of them are unlabeled at this time.
I have been able to figure out what most of them do, but I am an advanced screen reader user and the process was tedious at best.

**Comment by balloob** [comment]
I must admit that i have never tested this with a screenreader. I will put this on my to do list to address.

**Comment by balloob** [comment]
Do you have a screen reader that you recommend to test with?

**Comment by justweb1** [comment]
I believe this will primarily be labeling everything with aria-label. https://www.google.com/url?sa=t&source=web&rct=j&url=%23&ved=0ahUKEwjZlcDyptDTAhVmi1QKHaaxB54Qxa8BCCAwAg&usg=AFQjCNF1sHlk2z9QIx28jtvfnGP_UlkifA&sig2=pRgbDqrfpWCheies57FebQ

**Comment by justweb1** [comment]
Here is a quick video on accessibility in Polymer. https://youtu.be/Lktz1KXbTOU

**Comment by jfayre** [comment]
Hi,

Thanks so much for looking into this!

I primarily use Windows and IOS devices. On Windows, the screen reader I use is called NVDA:

http://www.nvaccess.org

The other one I use is JAWS for Windows, whci his not a free product:

http://www.freedomscientific.com

 

If you have never interacted with a screen reader before, check this article:

http://webaim.org/articles/nvda/

I’m a very experienced screen reader user. Feel free to ask any questions you might have.

**Comment by jfayre** [comment]
Hello,
Do we have any update on this? Last night, I installed hass.io and am still seeing the same accessibility issues.

**Comment by balloob** [comment]
Sorry, it's still on my to do list 😞

**Comment by zsarnett** [comment]
\move ui-schema
